### PR TITLE
feat: revokeTrustedManagerPermission method for sc

### PR DESCRIPTION
**Pull request by samvel33651**
added frevokeTrustedManagerPermission method

### CODE REVIEW

**Comment by skambalin** [comment]
@samvel33651 can you please add a test for this method somewhere in [this describe](https://github.com/Cerebellum-Network/cere-ddc-sdk-js/blob/main/tests/SmartContract.spec.ts#L53)

**Review by skambalin — APPROVED** [review]

File: tests/SmartContract.spec.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ describe('Smart Contract', () => {
             await adminContract.grantTrustedManagerPermission(user.address);
         });
 
+        test(' revoke trusted manager permission', async () => {
```
Comment (skambalin) [inline tests/SmartContract.spec.ts:62]: Let's remove the space

```suggestion
        test('revoke trusted manager permission', async () => {
```

File: tests/SmartContract.spec.ts [hunk 2]
```diff
@@ -59,6 +59,10 @@ describe('Smart Contract', () => {
...
+            await adminContract.revokeTrustedManagerPermission(admin.address);
```
[the remark this answers is not in the document]
Comment (samvel33651) [reply tests/SmartContract.spec.ts:63]: In this case we need to revoke cluster manager permissions, I not quiet sure that user has cm permissions.

[the remark this answers is not in the document]
Comment (skambalin) [reply tests/SmartContract.spec.ts:63]: `user` is granted the permission on [the previous step](https://github.com/Cerebellum-Network/cere-ddc-sdk-js/pull/125/files#diff-43fca3b81b39bb66c437efdadc0cccca5218669e80e7ca3386f237814b741336R58).

[the remark this answers is not in the document]
Comment (samvel33651) [reply tests/SmartContract.spec.ts]: @skambalin I have added `clusterManager` user for this test.

**Pull request merged**